tests/project_directory_test.php

```diff
@@ -107,4 +107,20 @@ public function testGetDirectoryAvoidPluginsDirectory()
         $this->assertInstanceOf('\\Dxw\\Whippet\\ProjectDirectory', $result->unwrap());
         $this->assertEquals($dir, $result->unwrap()->__toString());
     }
+
+    public function testGetDirectoryWithEnv()
+    {
+        $dir = $this->getDir();
+
+        putenv('WHIPPET_PROJECT_DIRECTORY=/etc');
```

Comment:
Another thought: would this override the user's local environment? So if someone had this var set up for their local dev (probably unlikely) then the test would override that? 

Possibly something to file under 'maybe possible, but not likely enough to worry about'